pyscriptjs/src/plugins/stdiodirector.ts

```diff
@@ -0,0 +1,28 @@
+import { Plugin } from "../plugin";
+import { TargetedStdio, StdioMultiplexer } from "../stdio";
+
+export class StdioDirector extends Plugin {
+    _stdioMultiplexer: StdioMultiplexer;
+
+    constructor(stdio: StdioMultiplexer) {
+        super()
+        this._stdioMultiplexer = stdio
+    }
+
+    beforePyScriptExec(runtime: any, src: any, PyScriptTag): void {
+        if (PyScriptTag.hasAttribute("output")){
+            const target_id = PyScriptTag.getAttribute("output")
```

Comment:
Small nit feel free to ignore: Perhaps the var should be camelcase

```suggestion
            const targetId = PyScriptTag.getAttribute("output")
```